### PR TITLE
add tag for BVT on canary testing

### PR DESCRIPTION
**Pull request by yahliu**
Signed-off-by: yahliu@redhat.com

Relate with the task [13807](https://github.com/open-cluster-management/backlog/issues/13807), the requirement is needing to define automated test cases to be run in the one-hour BVT canary. Add `@bvt` tag to collect the basic cases to verify each policy controller in BVT canary pipeline.  
fyi @drkho

### CODE REVIEW

File: tests/cypress/tests/CertPolicy_governance.spec.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ if (Cypress.env('MANAGED_CLUSTER_NAME') !== undefined) {
 }
 
 
-describeT('@extended Setup - create a certificate expiring soon', () => {
+describeT('@bvt Setup - create a certificate expiring soon', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/CertPolicy_governance.spec.js:20]:
```suggestion
describeT('@extended @bvt Setup - create a certificate expiring soon', () => {
```

File: tests/cypress/tests/CertPolicy_governance.spec.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ describeT('@extended Setup - create a certificate expiring soon', () => {
 })
 
 
-describeT('@extended RHACM4K-2294 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
+describeT('@bvt RHACM4K-2294 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/CertPolicy_governance.spec.js:38]:
```suggestion
describeT('@extended @bvt RHACM4K-2294 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
```

File: tests/cypress/tests/CertPolicy_governance.spec.js [hunk 3]
```diff
@@ -119,7 +119,7 @@ describeT('@extended RHACM4K-2294 - GRC UI: [P1][Sev1][policy-grc] - Certificate
 })
 
 
-describeT('@extended RHACM4K_1205 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
+describeT('@bvt RHACM4K-1205 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/CertPolicy_governance.spec.js:122]:
```suggestion
describeT('@extended @bvt RHACM4K-1205 - GRC UI: [P1][Sev1][policy-grc] - CertificatePolicy governance', () => {
```

File: tests/cypress/tests/CertPolicy_governance.spec.js [hunk 4]
```diff
@@ -203,7 +203,7 @@ describeT('@extended RHACM4K_1205 - GRC UI: [P1][Sev1][policy-grc] - Certificate
 })
 
 
-describeT('@extended Cleanup - delete a certificate and an issuer', () => {
+describeT('@bvt Cleanup - delete a certificate and an issuer', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/CertPolicy_governance.spec.js:206]:
```suggestion
describeT('@extended @bvt Cleanup - delete a certificate and an issuer', () => {
```

File: tests/cypress/tests/IamPolicy_governance.spec.js [hunk 5]
```diff
@@ -5,14 +5,14 @@
 import { describeT } from '../support/tagging'
 import { test_genericPolicyGovernance, test_applyPolicyYAML } from '../support/tests'
 
-describeT('@extended GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - setup', () => {
+describeT('@bvt GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - setup', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/IamPolicy_governance.spec.js:8]:
```suggestion
describeT('@extended @bvt GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - setup', () => {
```

File: tests/cypress/tests/IamPolicy_governance.spec.js [hunk 6]
```diff
@@ -5,14 +5,14 @@
...
   test_applyPolicyYAML('IamPolicy_governance/policy-clusterrolebinding-setup_raw.yaml')
 })
 
-describeT('@extended RHACM4K-1719 - GRC UI: [P1][Sev1][policy-grc] IamPolicy governance', () => {
+describeT('@bvt RHACM4K-1719 - GRC UI: [P1][Sev1][policy-grc] IamPolicy governance', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/IamPolicy_governance.spec.js:12]:
```suggestion
describeT('@extended @bvt RHACM4K-1719 - GRC UI: [P1][Sev1][policy-grc] IamPolicy governance', () => {
```

File: tests/cypress/tests/IamPolicy_governance.spec.js [hunk 7]
```diff
@@ -5,14 +5,14 @@
...
   test_genericPolicyGovernance('IamPolicy_governance/policy-config.yaml', 'IamPolicy_governance/violations-inform.yaml', 'IamPolicy_governance/violations-inform.yaml')
 })
 
-describeT('@extended GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - cleanup', () => {
+describeT('@bvt GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - cleanup', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/IamPolicy_governance.spec.js:16]:
```suggestion
describeT('@extended @bvt GRC UI: [P1][Sev1][policy-grc] IamPolicy governance - cleanup', () => {
```

File: tests/cypress/tests/Namespace_governance.spec.js [hunk 8]
```diff
@@ -5,12 +5,12 @@
 import { describeT } from '../support/tagging'
 import { test_genericPolicyGovernance, test_applyPolicyYAML } from '../support/tests'
 
-describeT('@extended GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - setup', () => {
+describeT('@bvt GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - setup', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/Namespace_governance.spec.js:8]:
```suggestion
describeT('@extended @bvt GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - setup', () => {
```

File: tests/cypress/tests/Namespace_governance.spec.js [hunk 9]
```diff
@@ -5,12 +5,12 @@
...
     test_applyPolicyYAML('Namespace_governance/Namespace_specification_setup_policy_raw.yaml')
 })
-describeT('@extended RHACM4K-1725 - GRC UI: [P1][Sev1][policy-grc] Namespace policy governance', () => {
+describeT('@bvt RHACM4K-1725 - GRC UI: [P1][Sev1][policy-grc] Namespace policy governance', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/Namespace_governance.spec.js:11]:
```suggestion
describeT('@extended @bvt RHACM4K-1725 - GRC UI: [P1][Sev1][policy-grc] Namespace policy governance', () => {
```

File: tests/cypress/tests/Namespace_governance.spec.js [hunk 10]
```diff
@@ -5,12 +5,12 @@
...
     test_genericPolicyGovernance('Namespace_governance/policy-config.yaml', 'Namespace_governance/violations-inform.yaml', 'Namespace_governance/violations-enforce.yaml')
 })
-describeT('@extended GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - clean up', () => {
+describeT('@bvt GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - clean up', () => {
```
Comment (dhaiducek) [inline tests/cypress/tests/Namespace_governance.spec.js:14]:
```suggestion
describeT('@extended @bvt GRC UI: [P1][Sev1][policy-grc] Namespace policy governance - clean up', () => {
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
The `@extended` tags need to be preserved for current uses. However, you should be able to add the space-separated `@bvt` tag to it.

**Review by dhaiducek — APPROVED** [review]
LGTM!

**Comment by yahliu** [comment]
@dhaiducek I rebased codes due to conflict，please review it again. Thanks!

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**